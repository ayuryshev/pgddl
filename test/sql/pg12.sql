\pset null _null_
\pset format unaligned

SET client_min_messages = warning;
SET ROLE postgres;

create table tab_generated (
       a integer,
       b integer,
       c integer generated always as (a+b) stored
);

select ddlx_script('tab_generated');
