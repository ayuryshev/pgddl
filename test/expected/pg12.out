\pset null _null_
\pset format unaligned
SET client_min_messages = warning;
SET ROLE postgres;
create table tab_generated (
       a integer,
       b integer,
       c integer generated always as (a+b) stored
);
select ddlx_script('tab_generated');
ddlx_script
BEGIN;

/*
DROP TABLE tab_generated;
*/

-- Type: TABLE ; Name: tab_generated; Owner: postgres

CREATE TABLE tab_generated (
    a integer,
    b integer,
    c integer
);

COMMENT ON TABLE tab_generated IS NULL;
ALTER TABLE tab_generated OWNER TO postgres;

ALTER TABLE tab_generated ALTER c SET DEFAULT (a + b);


END;

(1 row)
